src/clustering/bkmeans_internal.h

```diff
@@ -145,7 +145,7 @@ class BKmeansInternal : public IBKmeansInternal {
 
         // critical section
 #pragma omp parallel for
-        for (unsigned int i = 0; i < data.size(); i++) {
+        for (int i = 0; i < static_cast<int>(data.size()); i++) {
```

Comment:
Thank you for your fix! Could you plz clarify the reason of this modification ?

Reply:
OpenMP "2.0" C/C++ API specification requires that index variables in `for` to be signed.
https://stackoverflow.com/questions/2820621/why-arent-unsigned-openmp-index-variables-allowed

Reply:
Thank you for the info. LGTM!